{"uuid": "9580a075-02c6-4f82-9fae-9fc1b64ca587", "children": ["670622bf-9405-4bc1-9de7-7992615e7910"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_RESET\n  (Session info: headless chrome=114.0.5735.134)\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x0116A813+48355]\n\t(No symbol) [0x010FC4B1]\n\t(No symbol) [0x01005358]\n\t(No symbol) [0x01001B33]\n\t(No symbol) [0x00FF8CA5]\n\t(No symbol) [0x00FF9D0A]\n\t(No symbol) [0x00FF9003]\n\t(No symbol) [0x00FF8355]\n\t(No symbol) [0x00FF82EC]\n\t(No symbol) [0x00FF6F26]\n\t(No symbol) [0x00FF7688]\n\t(No symbol) [0x01006B52]\n\t(No symbol) [0x0105CF01]\n\t(No symbol) [0x0104A73C]\n\t(No symbol) [0x0105C922]\n\t(No symbol) [0x0104A536]\n\t(No symbol) [0x010282DC]\n\t(No symbol) [0x010293DD]\n\tGetHandleVerifier [0x013CAABD+2539405]\n\tGetHandleVerifier [0x0140A78F+2800735]\n\tGetHandleVerifier [0x0140456C+2775612]\n\tGetHandleVerifier [0x011F51E0+616112]\n\t(No symbol) [0x01105F8C]\n\t(No symbol) [0x01102328]\n\t(No symbol) [0x0110240B]\n\t(No symbol) [0x010F4FF7]\n\tBaseThreadInitThunk [0x764000C9+25]\n\tRtlGetAppContainerNamedObjectPath [0x778B7B4E+286]\n\tRtlGetAppContainerNamedObjectPath [0x778B7B1E+238]\n\n", "trace": "  File \"D:\\Credence Python Projects by Tushar Sir\\Automation_Credkart\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\Credence Python Projects by Tushar Sir\\Automation_Credkart\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\Credence Python Projects by Tushar Sir\\Automation_Credkart\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 909, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\Credence Python Projects by Tushar Sir\\Automation_Credkart\\testCases\\conftest.py\", line 30, in setup\n    driver.get(\"https://automation.credence.in/\")\n  File \"D:\\Credence Python Projects by Tushar Sir\\Automation_Credkart\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 355, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"D:\\Credence Python Projects by Tushar Sir\\Automation_Credkart\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"D:\\Credence Python Projects by Tushar Sir\\Automation_Credkart\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1687410191793, "stop": 1687410195894}], "start": 1687410191793, "stop": 1687410196070}